python/python-core/sequential-data-types-ii/python-lists.md

```diff
@@ -49,7 +49,7 @@ print(my_first_list)
 # ['bananas', 'oranges', 'pears']
 ```
 
-We can also use our `index` and `count` methods that we learnt about back in the `tuple` days on lists.
+> We can also use our `index` and `count` methods that we learnt about back in the [Tuple Methods lesson](https://app.enki.com/insights/tuple-methods).
```

Comment:
should we use "learned" here since it's [the preferred US spelling](https://www.grammarly.com/blog/learned-learnt/)?
```suggestion
> We can also use our `index` and `count` methods that we learned about back in the [Tuple Methods lesson](https://app.enki.com/insights/tuple-methods).
```

Reply:
I think this might be a content-wide change. I've noticed that we use "learnt" in multiple places :thinking:

Reply:
hmm, good point